src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
-    <description>NPSP Getting Started Page section items</description>
+    <description>NPSP Get Started Page section items</description>
     <fields>
         <fullName>Description_Label__c</fullName>
-        <description>The Description of a section item on the Get Started page.</description>
+        <description>The API Name for Description label of this item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</description>
```

Comment:
@fparodi-salesforce Can you add the word "the" before Description so that it reads: "The API name for the Description..."

Please make this change to the inlineHelpText too. Thanks!